Sources/TuistCache/Mappers/Graph/FocusTargetsGraphMappers.swift

```diff
@@ -5,19 +5,33 @@ import TuistGraph
 
 /// `FocusTargetsGraphMappers` is used to filter out some targets and their dependencies and tests targets.
 public final class FocusTargetsGraphMappers: GraphMapping {
+    // When specified, if includedTargets is empty it will automatically include all targets in the test plan
+    let testPlan: String?
     /// The targets name to be kept as non prunable with their respective dependencies and tests targets
     let includedTargets: Set<String>
+    let excludedTargets: Set<String>
 
-    public init(includedTargets: Set<String>) {
+    public init(
+        testPlan: String? = nil,
+        includedTargets: Set<String>,
+        excludedTargets: Set<String> = []
+    ) {
```

Comment:
avoid defaults in internal methods, they just risk you forget passing parameters
```suggestion
    public init(testPlan: String?, includedTargets: Set<String>, excludedTargets: Set<String>) {
```